packages/ibm-products/src/components/TagOverflow/TagOverflowPopover.js

```diff
@@ -0,0 +1,198 @@
+//
+// Copyright IBM Corp. 2024, 2024
+//
+// This source code is licensed under the Apache-2.0 license found in the
+// LICENSE file in the root directory of this source tree.
+//
+
+import React, { useRef } from 'react';
+import PropTypes from 'prop-types';
+
+import cx from 'classnames';
+
+import { Link, Tag, Popover, PopoverContent } from '@carbon/react';
+import { useClickOutside } from '../../global/js/hooks';
+
+import { pkg } from '../../settings';
+
+const componentName = 'TagOverflowPopover';
+const blockClass = `${pkg.prefix}--tag-overflow-popover`;
+
+// Default values for props
+const defaults = {
+  allTagsModalSearchThreshold: 10,
+  overflowAlign: 'bottom',
+};
+
+export const TagOverflowPopover = React.forwardRef(
+  (
+    {
+      // The component props, in alphabetical order (for consistency).
+
+      allTagsModalSearchThreshold = defaults.allTagsModalSearchThreshold,
+      className,
+      onShowAllClick,
+      overflowAlign = defaults.overflowAlign,
+      overflowTags,
+      overflowType,
+      showAllTagsLabel,
+      popoverOpen,
+      setPopoverOpen,
+      // Collect any other property values passed in.
+      ...rest
+    },
+    ref
+  ) => {
+    const localRef = useRef();
+    const overflowTagContent = useRef(null);
+
+    useClickOutside(ref || localRef, () => {
+      if (popoverOpen) {
+        setPopoverOpen(false);
+      }
+    });
+
+    const handleShowAllTagsClick = (ev) => {
```

Comment:
i think as a convention we tend to use `evt` for event variable names. i would update references to events to just be `evt`

Reply:
Done